internal/modules/autopi/autopi.go

```diff
@@ -0,0 +1,192 @@
+package autopi
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/DIMO-Network/model-garage/pkg/autopi/status"
+
+	"github.com/DIMO-Network/model-garage/pkg/cloudevent"
+	"github.com/DIMO-Network/model-garage/pkg/convert"
+	"github.com/DIMO-Network/model-garage/pkg/vss"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/redpanda-data/benthos/v4/public/service"
+	"github.com/segmentio/ksuid"
+)
+
+const (
+	StatusEventType      = "com.dimo.device.status.v2"
+	FingerprintEventType = "zone.dimo.aftermarket.device.fingerprint"
+	DataVersion          = "v2"
```

Comment:
set v2 only, since we will not stream any old v1 payloads at all. All AP are migrated to v2.

Reply:
Let's pull this off the payload since we may feed v1 payloads in to DIS if we ever need to backport signals.